### PR TITLE
 [stable/kong] Cassandra, "wait-for-db" initContainer failed to start: Error: [postgres error] could not retrieve server_version: connection refused

### PR DESCRIPTION
**Pull request by dkozlov**
**What this PR does / why we need it:**
Fixed issue when **env.database=cassandra** and **cassandra.enabled=true**, but "wait-for-db" initContainer failed to start: Error: [postgres error] could not retrieve server_version: connection refused. So we explicitly set KONG_DATABASE variable for 'wait-for-db' initContainer as it was already done for main kong container.

```
kubectl get all -n kong
NAME                             READY     STATUS      RESTARTS   AGE
pod/kong-cassandra-0             1/1       Running     0          32m
pod/kong-cassandra-1             1/1       Running     0          30m
pod/kong-cassandra-2             1/1       Running     0          28m
pod/kong-kong-ddc97bf6f-lvmm7    0/1       Init:0/1    0          32m
pod/kong-kong-migrations-jx9vd   0/1       Completed   5          32m

NAME                      TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)                                        AGE
service/kong-cassandra    ClusterIP   None           <none>        7000/TCP,7001/TCP,7199/TCP,9042/TCP,9160/TCP   32m
service/kong-kong-admin   NodePort    10.233.9.69    <none>        8444:32712/TCP                                 32m
service/kong-kong-proxy   NodePort    10.233.2.163   <none>        8443:30707/TCP                                 32m

NAME                        DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/kong-kong   1         1         1            0           32m

NAME                                  DESIRED   CURRENT   READY     AGE
replicaset.apps/kong-kong-ddc97bf6f   1         1         0         32m

NAME                              DESIRED   CURRENT   AGE
statefulset.apps/kong-cassandra   3         3         32m

NAME                             DESIRED   SUCCESSFUL   AGE
job.batch/kong-kong-migrations   1         1            32m
```
```
kubectl exec -it kong-kong-ddc97bf6f-lvmm7 -c wait-for-db -n kong /bin/sh
/ # kong start
Error: [postgres error] could not retrieve server_version: connection refused
```
Thanks !

#### Which issue this PR fixes
Issue was introduced by following commit https://github.com/helm/charts/commit/0283024f05c30ae9ca6c4b7e50366d0834414878 (@shashiranjan84)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by shashiranjan84** [comment]
@dkozlov good catch and thanks for PR.

File: stable/kong/Chart.yaml [hunk 1]
```diff
@@ -10,5 +10,5 @@ maintainers:
 name: kong
 sources:
 - https://github.com/Kong/kong
-version: 0.5.0
-appVersion: 0.14.1
+version: 0.5.1
+appVersion: 0.14.2
```
Comment (hbagdi) [inline stable/kong/Chart.yaml]: Wait, why is the `appVersion` changed?
Only the version of the chart should be bumped up!

Comment (shashiranjan84) [reply stable/kong/Chart.yaml]: Right, totally overlooked that. @dkozlov would you please revert appVersion back to `0.14.1`

Comment (dkozlov) [reply stable/kong/Chart.yaml]: appVersion was changed to 0.14.1

Comment (dkozlov) [reply stable/kong/Chart.yaml]: Do I need to change chart version to 0.5.2?

Comment (shashiranjan84) [reply stable/kong/Chart.yaml]: Yes, as https://github.com/helm/charts/blob/master/stable/kong/Chart.yaml#L13

File: stable/kong/Chart.yaml [hunk 2]
```diff
@@ -11,4 +11,4 @@ name: kong
 sources:
 - https://github.com/Kong/kong
 version: 0.5.2
-appVersion: 0.14.1
+appVersion: 0.14.1
```
Comment (shashiranjan84) [inline stable/kong/Chart.yaml]: please add the new line as good practice

Comment (dkozlov) [reply stable/kong/Chart.yaml]: new line has been added

Comment (dkozlov) [reply stable/kong/Chart.yaml]: @shashiranjan84 which character I need to add in end of the line to pass these tests? Now I have added "\n" but tests were failed with `(error    too many blank lines (1 > 0)  (empty-lines))`

**Comment by dkozlov** [comment]
@shashiranjan84 could you please approve?
